seat.c

```diff
@@ -803,6 +849,14 @@ seat_create(struct cg_server *server, struct wlr_backend *backend)
 	seat->start_drag.notify = handle_start_drag;
 	wl_signal_add(&seat->seat->events.start_drag, &seat->start_drag);
 
+	seat->virtual_keyboard = wlr_virtual_keyboard_manager_v1_create(server->wl_display);
+	wl_signal_add(&seat->virtual_keyboard->events.new_virtual_keyboard, &seat->new_virtual_keyboard);
+	seat->new_virtual_keyboard.notify = handle_virtual_keyboard;
+
+	seat->virtual_pointer = wlr_virtual_pointer_manager_v1_create(server->wl_display);
+	wl_signal_add(&seat->virtual_pointer->events.new_virtual_pointer, &seat->new_virtual_pointer);
+	seat->new_virtual_pointer.notify = handle_virtual_pointer;
```

Comment:
Here we're creating one virtual-keyboard and virtual-pointer manager per seat. We shouldn't do this, instead we should only create one per server.